docs/pages/repo/docs/core-concepts/monorepos/running-tasks.mdx

```diff
@@ -270,6 +270,34 @@ be explicitly opted into via including `//#<task>` in the pipeline configuration
 some best-effort checking to produce an error in the recursion situations, but it is up to you to only
 opt in those tasks which don't themselves trigger a `turbo` run that would recurse.
 
+### Dependencies outside of a task
+  
+When your task has topological dependencies that are outside of that given task, you'll still want to enjoy the parallelism of Turborepo and ensure that your caching behavior is correct according to your code changes.
+  
+To demonstrate how to do this, let's say you have a set of workspaces to do a little bit of math: `add`, `subtract`, and `multiply`. You have tests in all three workspaces and you're using the `add` package in `subtract` and `multiply`. We have these packages set up as internal dependencies so they can be used directly in other workspaces.
+  
+It's time to run our tests and we have two requirements:
+  1. All tests run in parallel to keep things speedy
+  2. A change in a dependency should result in a cache miss
+
+To accomplish this, we can setup a pipeline like so:
+
+```jsonc
+{
+   "$schema": "https://turbo.build/schema.json",
+   "pipeline": {
+     "topo": {
+       "dependsOn": ["^topo"]
+     },
+     "test": {
+       "dependsOn": ["^topo"]
+     }
+   }
+ }
+```
+  
+In this pipeline, we create an intermediary dummy `topo` task. Since we don't have a `topo` command in our workspaces, the pipeline will go straight to running `test` scripts in parallel, meeting our first requirement. The second requirement will also be taken care of, falling back on Turborepo's default behavior of creating hashes for a task and it's dependencies.
```

Comment:
```suggestion
In this pipeline, we create an intermediary dummy `topo` task. Since we don't have a `topo` command in our workspaces, the pipeline will go straight to running `test` scripts in parallel, meeting our first requirement. The second requirement will also be taken care of, falling back on Turborepo's default behavior of creating hashes for a workspace task and it's dependencies as a tree.
```